### PR TITLE
Add linea support

### PR DESCRIPTION
**Pull request by Akaryatrh**
## Description

- Adding support for Linea mainnet, as it has been done for Linea testnet a few weeks ago.
- Also updating some urls for testnet

## Additional Information

- [x] I read the [contributing docs](/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md) (if this is your first contribution)

Your ENS/address:

### CODE REVIEW

File: packages/chains/src/linea.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  blockExplorers: {
+    default: {
+      name: 'Etherscan',
+      url: 'https://lineascan.build',
+    },
+  },
+  contracts: {
+    multicall3: {
+      address: '0x0',
+      blockCreated: 0,
```
Comment (Akaryatrh) [inline packages/chains/src/linea.ts:31]: These information are unknown for now (address should be identical to testnet, but let's wait for multicall contract deployment to fill them up)

**Comment by Akaryatrh** [comment]
Added multicall info https://explorer.linea.build/tx/0x369305bf856786a160151d3897de2540fc700cf3d6a289b954ca93460f1038b6

**Comment by liho00** [comment]
> Added multicall info https://explorer.linea.build/tx/0x369305bf856786a160151d3897de2540fc700cf3d6a289b954ca93460f1038b6

how you know this is the multicall info? 
I am learning it... Would you share me how to identify that as multicall address?

**Comment by Akaryatrh** [comment]
FYI explorer https://lineascan.build and bridge https://bridge.linea.build are now available 🙂
I then mark this PR as ready.

**Comment by liho00** [comment]
> > > Added multicall info https://explorer.linea.build/tx/0x369305bf856786a160151d3897de2540fc700cf3d6a289b954ca93460f1038b6
> > 
> > 
> > how you know this is the multicall info? I am learning it... Would you share me how to identify that as multicall address?
> 
> @liho00 All multicall contracts are deployed on the same address by the maintainer: https://github.com/mds1/multicall#deployments-and-abi Also contract is verified here: https://explorer.linea.build/address/0xcA11bde05977b3631167028862bE2a173976CA11/contracts#address-tabs

Appreciated your answers! I learned it. Thanks again.

**Pull request merged**